tests.lisp

```diff
@@ -5,6 +5,49 @@
 
 (in-package :split-sequence/tests)
 
+#+sbcl
```

Comment:
ABCL also supports extensible sequences, meaning that this test suite also becomes feasible to be run there, and that all `#+sbcl` should become `#+(or sbcl abcl)`.

Maybe it should be suggested to SBCL and ABCL people to push `:extensible-sequences` to features list instead?

Reply:
That would be a good idea, and in the meanwhile you can add that to trivial-features as well.

Reply:
Will do.